cmd/stats.go

```diff
@@ -138,3 +125,43 @@ func workflowStats(cfg config, opt options, isJobs bool) error {
 
 	return nil
 }
+
+func fetchWorkflowRuns(ctx context.Context, client *github.WorkflowStatsClient, cfg config, opt options) ([]*go_github.WorkflowRun, error) {
```

Comment:
`fetchWorkflowRuns` function could be called directly within the `client.FetchWorkflowRuns` method rather than being defined as a separate function. Is there a specific reason why it was defined as a function?

Reply:
> Is there a specific reason why it was defined as a function? 

Mainly to avoid bloating `workflowStats` function which is already fairly long. Feel free to adjust formatting to your liking.